web/src/main/resources/applicationContext-proxy.xml

```diff
@@ -9,6 +9,21 @@
      http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
      http://www.springframework.org/schema/mvc http://www.springframework.org/schema/mvc/spring-mvc.xsd">
 
+    <bean id="propertyPlaceholderConfigurer" class="org.springframework.beans.factory.config.PropertyPlaceholderConfigurer">
```

Comment:
I wasn't familiar with this file before.

Reply:
I borrowed the ideas from other application context. From what I understand, the web does not rely on the core module so the properties will not be injected unless you specify the property files in the web xml